LMS/Landing/Repositories/LandingRepository.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace LMS\Landing\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use LMS\Courses\Models\Course;
+use LMS\User\Models\User;
+
+class LandingRepository
+{
+    public function retrieveData(): array
+    {
+        return [
+            'achievements' => [
+                'students' => $this->getStudents(),
+                'contentHours' => $this->getCourseTotalTime(),
+                'watchedLessons' => $this->getTotalWatchedLessons()
+            ],
+            'courses' => [
+                'free' => $this->getFreeCourseInformation(),
+                'paid' => $this->getPaidCourseInformation()
+            ]
+        ];
+    }
+
+    private function getFreeCourseInformation(): array
+    {
+        $model = new Course();
+        $courseTime = 0;
+        foreach ($model->where('paid', false)->get() as $value) {
+            $courseTime += $value->duration;
+        }
+
+        return [
+            'count' => $model->where('paid', false)->count(),
+            'time' => gmdate('H:i:s', $courseTime),
+            'support' => '❌'
+        ];
+    }
+
+    private function getPaidCourseInformation(): array
+    {
+        $model = new Course();
+        $courseTime = 0;
+        foreach ($model->where('paid', true)->get() as $value) {
+            $courseTime += $value->duration;
+        }
+
+        return [
+            'count' => $model->where('paid', true)->count(),
+            'time' => gmdate('H:i:s', $courseTime),
+            'support' => '✅'
+        ];
+    }
+
+    private function getStudents(): int
+    {
+        return User::count();
+    }
+
+    private function getCourseTotalTime(): string
+    {
+        $model = new Course();
+        $courseTime = 0;
+
+        foreach ($model->where('paid', true)->get() as $value) {
```

Comment:
Estava lendo isso na [doc aggregates database](https://laravel.com/docs/8.x/queries#aggregates) do laravel, não sei se cabe neste exemplo mas ao  invés de fazer esse for each e percorrer os valores, realizar a soma e afins você pode retornar isso diretamente utilizando o `sum`
```suggestion
       $courseTime = $model->where('paid', true)->sum('duration');
```

Reply:
TLDR: se o atributo vier de um appends, você não pode usar aggregator.

Como esse atributo vem de um appends, você não consegue fazer a relação no SQL (que é o que as funções agregadores fazem). A ideia dos aggregators é sempre usar funções nativas do SQL pra fazer o trabalho, porém pra isso rolar o dado precisa estar na tabela.

Reply:
Tranquilo, não sabia sobre isso! 

Valeu por explicar, Dan!